decidim-core/app/presenters/decidim/admin_log/category_presenter.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Decidim
+  module AdminLog
+    # This class holds the logic to present a `Decidim::Category`
+    # for the `AdminLog` log.
+    #
+    # Usage should be automatic and you shouldn't need to call this class
+    # directly, but here's an example:
+    #
+    #    action_log = Decidim::ActionLog.last
+    #    view_helpers # => this comes from the views
+    #    CategoryPresenter.new(action_log, view_helpers).present
+    class CategoryPresenter < Decidim::Log::BasePresenter
+      private
+
+      def diff_fields_mapping
+        {
+          name: :i18n,
+          description: :i18n,
+          weight: :integer
```

Comment:
Can we also add the `parent_id` here to track changes against the parent category changes?